core/graph/src/test/java/uk/gov/gchq/gaffer/integration/operation/named/cache/NamedOperationCacheIT.java

```diff
@@ -89,36 +91,11 @@ public void after() throws CacheOperationException {
     }
 
     @Test
-    public void shouldWorkUsingHashMapServiceClass() throws OperationException, CacheOperationException {
-        reInitialiseCacheService(HashMapCacheService.class);
-        runTests();
-    }
-
-    private void reInitialiseCacheService(final Class clazz) throws CacheOperationException {
-        CacheServiceLoader.initialise(clazz.getCanonicalName());
-        CacheServiceLoader.getDefaultService().clearCache(CACHE_NAME);
-    }
-
-    private void runTests() throws OperationException, CacheOperationException {
-        shouldAllowUpdatingOfNamedOperations();
-        after();
-        shouldAllowUpdatingOfNamedOperationsWithAllowedUsers();
-        after();
-        shouldAllowReadingOfNamedOperationsUsingAdminAuth();
-        after();
-        shouldAllowUpdatingOfNamedOperationsUsingAdminAuth();
-        after();
-        shouldBeAbleToAddNamedOperationToCache();
-        after();
-        shouldBeAbleToDeleteNamedOperationFromCache();
-    }
-
-
-    private void shouldBeAbleToAddNamedOperationToCache() throws OperationException {
+    public void shouldBeAbleToAddNamedOperationToCache() throws OperationException {
         // given
         GetAllNamedOperations get = new GetAllNamedOperations.Builder().build();
         final Store store = mock(Store.class);
-        given(store.getProperties()).willReturn(properties);
+        given(store.getProperties()).willReturn(PROPERTIES);
```

Comment:
Not sure why the store needs to be mocked here when you could use the STORE variable above which already does this. Additionally, as this given statement is already in the BeforeAll I am not sure why it has been included in each test again?

Reply:
Thank you for your helpful reviews. I have addressed your comments by removing the duplicated GetAllNamedOperations creation code and using the STORE mock variable directly. Please let me know if there are any other improvements I can make. I appreciate you taking the time to review my changes.